connector.go

```diff
@@ -45,6 +45,7 @@ type Connector interface {
 	ExecuteAsync(expr string, args interface{}) *Future
 
 	NewPrepared(expr string) (*Prepared, error)
+	NewStream() *Stream
```

Comment:
It may be a good idea to return (*Stream, error) to act same for all cases (NewPrepared, get a response). Also it may be useful in the future.

No pressure, feel free to ignore.